servers/Azure.Mcp.Server/TROUBLESHOOTING.md

```diff
@@ -212,6 +213,36 @@ Clear Node Modules Cache
 
 - npm cache clean --force
 
+### MCP Tools That Require Additional Input Fail Silently
+
+The **Elicitation** feature in VS Code lets MCP tools request user input through interactive prompts during execution. If elicitation is not supported, affected tools may fail without showing prompts or may return errors about client compatibility. Updating VS Code usually resolves the issue.
+
+#### Requirements
+Elicitation is supported starting with **VS Code version 1.102 or newer** (released June 2025).
+
+#### Symptoms
+When elicitation isn't supported, you may experience:
+- MCP tools that need user input fail without explanation
+- Missing interactive prompts when tools request additional information
+- Error messages indicating elicitation is unsupported by the client
+
+![Elicitation error message](elicitation_not_supported.png)
```

Comment:
@xiangyan99 @JonathanCrd it would have been better to put this in a sub-folder than littering the root directory

Reply:
Addressed in https://github.com/microsoft/mcp/pull/851